react/features/breakout-rooms/functions.ts

```diff
@@ -40,25 +45,42 @@ export const getRoomsInfo = (stateful: IStateful) => {
 
     // only main roomn
     if (!breakoutRooms || Object.keys(breakoutRooms).length === 0) {
+        // filter out hidden participants
+        const conferenceParticipants = conference?.getParticipants().filter(participant => !participant._hidden);
```

Comment:
Other than the lint error, please use the `isHidden()` helper instead of accessing the `_hidden` property directly.